terra_layer/style.py

```diff
@@ -365,7 +423,15 @@ def boundaries_round(boundaries, scale=2):
     )
 
 
-def gen_legend_circle(min, max, size, color):
+def gen_legend_circle(
+    min,
+    max,
+    size,
+    color,
+    include_no_value,
+    no_value_circle_radius=DEFAULT_NO_VALUE_CIRCLE_RADIUS,
+    no_value_color=DEFAULT_NO_VALUE_FILL_COLOR,
+):
     """
     Generate a circle legend.
     """
```

Comment:
You can't set dict as default value directly.

```suggestion
def gen_legend_circle(
    min,
    max,
    size,
    color,
    include_no_value,
    no_value_circle_radius=DEFAULT_NO_VALUE_CIRCLE_RADIUS,
    no_value_color=None,
):
    """
    Generate a circle legend.
    """
    no_value_color= no_value_color or dict(DEFAULT_NO_VALUE_FILL_COLOR)
```

Reply:
May be we can remove default here as their are defined before...

Reply:
OK